File: SGEM/ClientApp/src/app/reportes/reporte.ts
export interface IReporte {
  id: number;
  titulo: string;
  tipoReporte: string;
  flag: number;
  items: IReporteItem[];
}

export interface IReporteItem {
  id?: number;
  idReporte?: number;
  nombre?: string;
  valor?: string;
}

File: SGEM/Models/Reporte.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SGEM.Models
{
    public class Reporte: Base
    {
        public string Titulo { get; set; }
        public string TipoReporte { get; set; }
        public int Flag { get; set; }
        public List<ReporteItem> Items { get; set; }

        public Reporte()
        {
            Items = new List<ReporteItem>();
        }
    }
}
